bin/dustc

```diff
@@ -1,82 +1,120 @@
 #!/usr/bin/env node
 
-var path = require('path'),
-    fs = require('fs'),
-    dust = require('../lib/server');
+var cli = require('cli').enable('glob', 'version'),
+    dust = require('../lib/server.js'),
+    path = cli.native.path,
+    eol = cli.native.os.EOL;
 
-var ARG_REGEX = /^--?([a-z][0-9a-z-]*)(?:=([^\s]+))?$/i,
-    args = process.argv.slice(1),
-    name = null;
+cli.setApp('dustc', require('../package.json').version);
 
-args = args.filter(function(arg) {
-    var match = arg.match(ARG_REGEX),
-        val;
+cli.setUsage(cli.app + ' [options] [path1 [path2 path3...]]\n\n' +
```

Comment:
Yep, new compiler. Had to kill the old syntax to allow it to work like a shell script.

`dustc input output` doesn't work anymore.

That's why I waited for a new minor version.